src/Main.js

```diff
@@ -1,15 +1,24 @@
 import {Component} from 'react';
 import HornedBeast from './HornedBeast.js';
-import Goat from './images/goat.jpeg';
-import WeirdGoat from './images/weirdGoat.jpeg';
+import data from './data.json';
+import {Container, Row, Col} from 'react-bootstrap';
 
 class Main extends Component {
   render() {
+    let newBeasts = [];
+    data.forEach(function(beast) {
+      let newBeast = <HornedBeast title={beast.title} imageUrl={beast.image_url} description={beast.description}></HornedBeast>;
+      let newBeastCol = <Col xs={12} md={6} lg={4}>{newBeast}</Col>;
+      newBeasts.push(newBeastCol);
+    }
+    );
+
```

Comment:
Good work on the lab!

For this, try using map instead :) Also, you will need to add your `key` as a prop too. Check your chrome dev tools in the `console` section. You will see React throwing error of : `Each child in a list should have a unique "key" prop.`